internal/rollup/rollup_bundle.bzl

```diff
@@ -611,10 +611,19 @@ ROLLUP_ATTRS = {
         It is sufficient to load one of these SystemJS boilerplate/entry point
         files as a script in your HTML to load your application""",
     ),
+    "auto_additional_entry_points": attr.bool(
+        doc = """Set this to True instead of specifying additional_entry_points to
```

Comment:
add a link to https://rollupjs.org/guide/en#code-splitting

and "code splitting is always enabled when additional_entry_points is non-empty"